src/components/pages/AboutPage/TeamMember.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '../../../utils/fontAwesome';
+import '../../../css/pages/AboutPage.css';
+
+class TeamMember extends React.Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    favMovie: PropTypes.string.isRequired,
+    favShow: PropTypes.string.isRequired,
+    favGenre: PropTypes.string.isRequired,
+    imgUrl: PropTypes.string.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showInfo: false
+    };
+  }
+
+  showInfo = () => {
+    this.setState({
+      showInfo: true
+    });
+  };
+
+  hideInfo = () => {
+    this.setState({
+      showInfo: false
+    });
+  };
+
+  render() {
+    const { name, title, favMovie, favShow, favGenre, imgUrl } = this.props;
+    const { showInfo } = this.state;
+    return (
+      <div className="avatar-size-container">
+        <div className="avatar-container">
+          <div className="avatar-item">
+            <img src={imgUrl} alt={name} className="avatar-image" />
+          </div>
+          <h3 className="avatar-name">{name}</h3>
+          <h5 className="avatar-role">
+            {title} <br />
+            Software Engineer
+          </h5>
+          {showInfo ? (
+            <div className="show-info fade-in">
+              <p className="avatar-movie">
+                Favorite Movies{' '}
+                <span role="img" aria-label="popcorn">
+                  🍿
+                </span>
+                : {favMovie}
+              </p>
+              <p className="avatar-show">
+                Favorite Show{' '}
+                <span role="img" aria-label="television set">
+                  📺
+                </span>
+                : {favShow}
+              </p>
+              <p className="avatar-genre"> Favorite Genre: {favGenre}</p>
+              <button
+                type="button"
+                className="show-less-button"
+                onClick={this.hideInfo}
+              >
+                <FontAwesomeIcon icon="angle-up" />
+              </button>
+            </div>
+          ) : (
+            <button
+              type="button"
+              className="show-more-button"
```

Comment:
Another benefit of having just a toggle function is that you could actually just have the more/less button actually be the same button, instead of having two. Basically instead of having it be part of the stuff that displays or doesn't, it always displays and just has a class that determines whether it should be rotated or not, based on if the info is shown. That's how the button works on the expanding move feed item.

```suggestion
              className=`toggle-info-button ${showInfo ? 'open' : ''}`
```

Then in the CSS:
```
.toggle-info-button.open {
  transform: rotate(180deg);
}